src/uct/ib/ud/base/ud_ep.c

```diff
@@ -1818,8 +1818,9 @@ ucs_status_t uct_ud_ep_invalidate(uct_ep_h tl_ep, unsigned flags)
     return UCS_OK;
 }
 
-int uct_ud_ep_is_same_dest_ep(const uct_ud_ep_t *ep,
-                              const uct_ep_is_connected_params_t *params)
+int uct_ud_ep_is_connected_to_addr(const uct_ud_ep_t *ep,
+                                   const uct_ep_is_connected_params_t *params,
+                                   uint32_t qp_num, uint32_t dct)
```

Comment:
"dct" is not right name for this parameter. it's UD transport and not DC.
The field is called "dqp_dct" in the PRM because it could be either Dest QP Number or DC Target number. For UD case, it's a QP number.